### ISSUE TITLE
Issue with receiving and sending mail to my main character

### ISSUE THREAD

**Issue by cottonz15** [slowtorta/turtlewow-bug-tracker#6282]
**Please, provide a clear description what the bug is:**
Since about a month or so ago, I have not been able to send my main toon (Dotsforu) more than 3 items of mail. I will get a error message saying not too send too much or its full or something. This only happens with one character and none of my alts. I have also tested with guildies (all level 60) and they cannot send me more than a few different mail and come across the same error. 

**Steps to reproduce the behavior with as much detail as possible:**

1. Send more than 3-4 mails to my main (Dotsforu)
2.  You will receive error message and not be able too.

**Expected behavior. Describe how it should work:**
Multiple mail items should be able to be sent to my main toon without any errors, from lvl 60s or otherwise.

**Comment by EyeOfTheBeast** [comment]
Is your mailbox full?

**Comment by cottonz15** [comment]
No, theres is nothing in my mailbox or at worst, only three items are in my mailbox. I have gone over this many times and tried to figure out what the problem is to no avail.

**Comment by cottonz15** [comment]
Just to add, I cant even send gold to my main.

**Comment by GrizB37** [comment]
Make sure subject is more than 1 or 2 words.

**Comment by schanzee** [comment]
https://github.com/slowtorta/turtlewow-bug-tracker/issues/6251#issuecomment-1943946031
This maybe? I can imagine this happening a lot if people keep answered mail in their inbox

**Comment by cottonz15** [comment]
> [#6251 (comment)](https://github.com/slowtorta/turtlewow-bug-tracker/issues/6251#issuecomment-1943946031)
> This maybe? I can imagine this happening a lot if people keep answered mail in their inbox

See this is what I thought it might be but it has been longer than 30 days now and I have checked my alt and my main and nothing.
